packages/mobile/src/dappkit/dappkit.ts

```diff
@@ -9,6 +9,7 @@ import {
 } from '@celo/utils/lib/dappkit'
 import { call, select, takeLeading } from 'redux-saga/effects'
 import { e164NumberSelector } from 'src/account/selectors'
+import { e164NumberToSaltSelector } from 'src/identity/reducer'
```

Comment:
nothing to do here, but we really need to rename this `salt` reference throughout the codebase.
https://github.com/celo-org/celo-monorepo/issues/4758